server/services/datacommons.py

```diff
@@ -327,6 +327,13 @@ def get_event_collection_date(event_type, affected_place):
   })
 
 
+def nl_helloworld():
+  """fetch nl hello world data.
+  """
+  url = cfg.NL_ROOT
```

Comment:
@Fructokinase what would be a good way to put this in discovery.py? Also I wonder if we should complete the work to have mixer service (and now with nl service)